python/perf-kernels/flash-attention.py

```diff
@@ -979,6 +1034,13 @@ def test_op_fwd(Z, H, N_CTX_Q, N_CTX_K, D_HEAD, causal, dtype=torch.float16):
     if causal:
         input_metadata.need_causal()
 
+    if use_alibi:
+        # for n heads the set of slopes is the geometric sequence that starts 2^(-8/n)
+        alibi_slopes = torch.tensor([2**(-8/H*i) for i in range(1, H+1)], dtype=torch.float32, device="cuda").repeat(Z, 1)
```

Comment:
I haven't read the paper thoroughly. OOC, Is this just a test value you used, or does the paper actually recommend this sequence?

Reply:
yes, this is from the paper